dbHelpers.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const db = require("./dbConfig");
+
+module.exports = {
+  insertUser,
+  findByUsername,
+  findUsers
+};
+
+function insertUser(user) {
+  return db("users").insert(user);
+}
+
+function findByUsername(username) {
+  return db("users").where("username", username);
+}
+function findUsers() {
+  return db("users").then(response => {
+    const mapped = response.map(x => {
```

Comment:
I'd use a more descriptive variable inside your map or a good old `item`, or `element` to keep it vanilla 👍